### PR TITLE
Migrating from Travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by dlockhart**
The major change here is the switch to use semantic-release for releasing new versions -- @klhuff let me know if you have concerns here that aren't explained by the README changes. We do have an incremental release action that's more similar to how this used to work (i.e. `[increment patch]`), but we prefer semantic-release for its other nice properties.

### CODE REVIEW

File: wct.conf.sauce.json [hunk 1]
```diff
@@ -5,13 +5,18 @@
       "idleTimeout": 360,
       "browsers": [
```
Comment (dlockhart) [inline wct.conf.sauce.json:6]: I merged these configs into a single file, as I couldn't think of an advantage to having them separate and didn't remember the original rationale. Previously, the tests would have run sequentially, whereas now Sauce will run them in parallel.

File: wct.conf.sauce.json [hunk 2]
```diff
@@ -5,13 +5,18 @@
...
         },
         {
           "browserName": "firefox",
-          "platform": "Windows 10",
+          "platform": "OS X 10.15",
+          "version": ""
+        },
+        {
+          "browserName": "safari",
+          "platform": "OS X 10.13",
```
Comment (dlockhart) [inline wct.conf.sauce.json:19]: I had to downgrade this to `10.13` as a test was failing on `10.15` -- likely something to investigate outside of this PR.

File: wct.conf.sauce.json (hunk 1 above)
Comment (svanherk) [reply wct.conf.sauce.json:6]: So I _think_ at one point the tests were so flakey and slow that we would randomly pick a config to run.  Or maybe we were _going_ to do that.  I can't remember, but I agree there's no need for them to be separate.

**Review by svanherk — APPROVED** [review]
Looks good to me!  I agree we should move this to `semantic-release`.  It'll be less confusing to keep the experience the same as all the component repos it uses (`core`, `organizations`, `enrollments`, `facet-filter-sort`, etc.)

**Pull request merged**